### ISSUE TITLE
Who wants to help organizing this? 

### ISSUE THREAD

**Issue by nanoparsec** [opencodetown/Chaos-Communication-Camp-2015#2]
I am happy to help :)

**Comment by cashewkernchen** [comment]
Not completely sure if I will be able to attend but I would be happy to support whenever possible (:

**Comment by finnp** [comment]
Hey :wave: 

At NodeSchool we were thinking about organizing a village as well (https://github.com/nodeschool/berlin/issues/18). I think it would be a good cool to do a joined Ruby+JavaScripttent/village.

@realtin and @lislis already said that they would like to help out as well.

How about doing an organizers meeting in May in Berlin? If we want to do workshops like RailsGirls or NodeSchool it would be good to think about the logistics early.

Best,
Finn

**Comment by jhilden** [comment]
Sounds good to me.

A Railsgirls Workshop would definitely be at the center of what we we are planing to organize.

I might be in Berlin the first week of May and would definitely be up for a meeting.

**Comment by finnp** [comment]
Hey everone, I created a doodle for the first week of may. It's 7pm everyday, but if this doesn't work out for someone, we can move it to another time as well: http://doodle.com/n4yua9tiparscf3y

**Comment by finnp** [comment]
@jhilden @nerdbabe Could you fill this out, too: http://doodle.com/n4yua9tiparscf3y :bee:

**Comment by jhilden** [comment]
Currently it looks like I won't be in Berlin next week.  But if possible I could join you via Skype or so for a little bit.

**Comment by finnp** [comment]
So we have a date :balloon: **It's May 4th at 7 pm**

@realtin @lislis So we still need a location with good wifi, that's still open at 7. Since it's Monday we might be able to just crush the OTS  ~~front end~~ python learning meetup at Co-Up or the Ruby study group(s).

@jhilden @nerdbabe

**Comment by Realtin** [comment]
Web Front-End Learners Meetup is on Wednesdays. Python & Ruby are on Mondays.

**Comment by skade** [comment]
Our office is always open, as long as there is one of the tenants there.

https://www.google.de/maps/place/Umspannwerk+Kreuzberg+GmbH/@52.4934903,13.4266164,16z/data=!4m5!1m2!2m1!1sumspannwerk!3m1!1s0x47a84fb457ec5c5b:0xcbc2c39110a0c047?hl=de

**Comment by finnp** [comment]
@skade Great, so let's meet at your office then.

Okay, everybody, here's the info again:

``` json
{
    "topic": "CCCamp planning",
    "time": "May 4th, 7pm",
    "location": {
       "name": "Asquera (Umspannwerk Kreuzberg)",
       "address": "Ohlauer Straße 43, 10999 Berlin"
    }
}
```

**Comment by johannhof** [comment]
Hey everyone, @finnp alerted me something's going on here and so I'll just join you at the Umspannwerk to plan stuff. See you there!

**Comment by skade** [comment]
@johannhof Sure, wanted to send you an email today anyways.

**Comment by sangyye** [comment]
I also would like to help :smile:

**Comment by finnp** [comment]
@sangyye If you have time on Monday, just come along :)

**Comment by sangyye** [comment]
@finnp Thanks, but I seems to have missed the information about the meeting being in Berlin, I live in Cologne and have to work, so its quite unlikely for me to attend this in person :(

**Comment by jhilden** [comment]
My Skype username is "jakobhilden".  Just call me whenever you are ready.

**Comment by Lelith** [comment]
hi there, is there still help needed? ping me :> would love to take care of #food

**Comment by skade** [comment]
@Lelith Everything is still open, so: definitely. We are committed to renting a tent, a large one possibly, but the prices are not yet known.

After that, we are interested in providing program - workshops, projects, ideas. Anything helps.

Food sounds like a great idea we didn't even speak about :)

**Comment by Lelith** [comment]
@skade there was an update on the rent a tent page. prices are there now :) and you have to commit a tent until the 20th of june. so we have to come up there quite quick now.

as i am a expierienced camper i also have some other gear already in place.

**Comment by skade** [comment]
Hm, chair and desk prices are not yet up. Do we want a 72 person tent? We'd have to allocate ~2.5k of budget then, which sounds doable.

**Comment by Lelith** [comment]
how big was the tent the last time? do we have couches and other seating (like bean bags org sth)?

**Comment by skade** [comment]
Wasn't there last time, but given that we want to get three groups into the tent, space seems like a nice thing to have.

**Comment by Lelith** [comment]
true, so +1 to space. let's move the tent discussion over to the tent issue :) so things keep organized

**Comment by anoukruhaak** [comment]
Hey all! we at Hackership are also contemplating setting up a village and would much prefer teaming up with all of you! How can we help?

**Comment by finnp** [comment]
@anoukruhaak That's great! :star2: One way would be to have someone come along at the next meeting https://github.com/rubytown/Chaos-Communication-Camp-2015/issues/13. That one is important, because we have to decide on the size of the tent. Otherwise helping out with fundraising would be great at this point.

**Comment by skade** [comment]
@anoukruhaak Welcome! Seems like we need to get a bigger tent :). On way to help is definitely to spread our fundraiser that can be found here (not just by retweet).

https://www.betterplace.org/en/projects/30076-an-assembly-on-chaos-communication-camp-providing-free-teaching

**Comment by anoukruhaak** [comment]
Will do! In fact, I think Hackership may be able to donate much if not all
of the outstanding amount. Let me discuss this with my co-organizers!

On 8 June 2015 at 03:07, Florian Gilcher notifications@github.com wrote:

> @anoukruhaak https://github.com/anoukruhaak Welcome! Seems like we need
> to get a bigger tent :). On way to help is definitely to spread our
> fundraiser that can be found here (not just by retweet).
> 
> https://www.betterplace.org/en/projects/30076-an-assembly-on-chaos-communication-camp-providing-free-teaching
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/rubytown/Chaos-Communication-Camp-2015/issues/2#issuecomment-109885877
> .

## 

_Time to become a better developer. Join Hackership in Berlin this summer,
or in Costa Rica this fall. Apply now http://hackership.org/apply!_

**Comment by anoukruhaak** [comment]
Will do! In fact, I think Hackership may be able to donate much if not all of the outstanding amount. Let me discuss this with my co-organizers!

**Comment by skade** [comment]
Hi @anoukruhaak,

That would be nice!

Please note that this is an amount for one of the tent sizes - if we can get a bigger one, great!

I'd prefer to leave the fundraiser running and keep funds that we commit to in private or donate them later. We can overshoot the budget if we want.

Let's talk about that in the organisers meeting.

(FWIW, asquera might also cover some outstanding cost)

Best,
Florian

**Comment by ralphtheninja** [comment]
I can help out with node related stuff, nodeshool, nodebots etc.

**Comment by KsenyaSerova** [comment]
Hi all, I would like to help with stuff.
How big is the tent by the way? Are you still fundraising for a bigger tent?

**Comment by finnp** [comment]
@KsenyaSerova Hey, sounds great. I also saw your mail and gonna reply to it now. Our tent is 96qm and we already ordered it. So the fundraising is more for other stuff for the camp now.

**Comment by lislis** [comment]
hey people! Sry I am so unresponsive right now but some things changed on my agenda and I'm not even sure if I can attend the camp at all :/  I'd like to help out wherever I can but that won't probably be much...

**Comment by finnp** [comment]
@lislis No worries. I feel like right now the organizing is working quite well. There are quite a few helping hands. Would be great to have you at the camp though! :cherry_blossom:

**Comment by lislis** [comment]
@finnp I'd like to be there, too! Let's see what happens.

**Comment by KsenyaSerova** [comment]
Does anyone knows when villages are turning up - a day in advance to set everything up, or earlier?

**Comment by max-mapper** [comment]
I'd like to help!

**Comment by sellibitze** [comment]
I'd like to help with Rust-related courses/workshops/introductions.

**Comment by skade** [comment]
@sellibitze \o/. Considering that the CCCamp is a rather flexible event, we decided not to pre-plan these activities too much, so just drop by and plan with us.

I'm not sure if we set a special appointment for that - you are always welcome and there will always be someone around the tent.

**Comment by sellibitze** [comment]
@skade All righty. Thanks. :) Grüße aus Göttingen!
